Formal2024/Homeworks/homework2.lean

```diff
@@ -0,0 +1,80 @@
+/-
+# Homework 1: Logic
+References: [MIL] Mathematics in Lean, [Tut] Tutorials project.
+Most of this homework is taken from [Tut04]
+-/
+
+import Mathlib.Data.Real.Basic
+import Mathlib.Tactic
+
+/-
+The next exercises use divisibility in ℤ (beware the ∣ symbol which is
+not ASCII).
+
+By definition, a ∣ b ↔ ∃ k, b = a*k, so you can prove a ∣ b using the
```

Comment:
```suggestion
By definition, `a ∣ b ↔ ∃ k, b = a * k`, so you can prove `a ∣ b` using the
```

Reply:
Since we haven't seen iff yet, I would change `↔` with `:=`.